<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="hp-PC" name="com.advertise.tests.TestSnooze" tests="1" failures="0" timestamp="12 Jan 2016 05:42:41 GMT" time="97.434" errors="1">
  <testcase name="testSnoozeIYForDesktop" time="97.434" classname="com.advertise.tests.TestSnooze">
    <error type="java.lang.AssertionError" message="No Ad Coverage !! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: No Ad Coverage !! expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at com.advertise.utils.TestCaseUtils.verifyResult(TestCaseUtils.java:68)
	at com.advertise.tests.TestSnooze.testSnooze(TestSnooze.java:189)
	at com.advertise.tests.TestSnooze.testSnoozeIYForDesktop(TestSnooze.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- testSnoozeIYForDesktop -->
</testsuite> <!-- com.advertise.tests.TestSnooze -->
